server/app/routes/api/v1/annotations/index.js

```diff
@@ -1,16 +1,23 @@
 'use strict';
 var router = require('express').Router();
-module.exports = router;
+var _ = require('lodash');
+
+const sequelizeHandlers = require( 'sequelize-handlers' );
+const bluebird = require( 'bluebird' );
+const utils = require('../../../utils');
+const {ensureAuthenticated, ensureIsAdmin, Credentials, respondWith404, _err, db} = utils;
```

Comment:
You don't need to define things you aren't using -- though I am guessing you may use them moving forward since we are at the beginning